testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.tests.js

```diff
@@ -6134,6 +6134,26 @@ QUnit.test("empty selection should be restored from state storing if selectedRow
     assert.deepEqual(dataGrid.getSelectedRowsData(), [], "getSelectedRowsData result");
 });
 
+QUnit.test("assign null to selectedRowKeys option unselect selected items", function(assert) {
```

Comment:
Is it possible to write this test in "datagrid/selection.tests.js"?
In this case, the test should take less time and will contains in a right place.

Reply:
It's not possible to place this test in "selection.test.js", I tried already.